docs/integrate/etl/influxdb.md

```diff
@@ -93,6 +89,51 @@ Verify that relevant data has been transferred to CrateDB.
 crash --host=cratedb --command="SELECT * FROM testdrive.demo;"
 :::
 
+## Cloud to Cloud
+
+If you're interested in migrating straight from [InfluxDB Cloud] to
+[CrateDB Cloud], the procedure is similar with small adjustments.
```

Comment:
On this page, being a technical guideline, we've adjusted our thinking to abstain from the "migration" jargon, and, attaching to the ETL acronym, talk about "data loading", or "data import/export" instead.

```suggestion
If you're interested in importing data from [InfluxDB Cloud] into
[CrateDB Cloud], the procedure is similar with small adjustments.
```